h2o-py/setup.py

```diff
@@ -3,14 +3,18 @@
 from codecs import open
 import os
 import shutil
-import h2o
 
 here = os.path.abspath(os.path.dirname(__file__))
 
 # Get the long description from the relevant file
 with open(os.path.join(here, 'DESCRIPTION.rst'), encoding='utf-8') as f:
     long_description = f.read()
 
+# Get the version from the relevant file
+with open(os.path.join(here, 'h2o/version.txt'), encoding='utf-8') as f:
+    version = f.read()
```

Comment:
Can we have default here, which will generate crippled version?

Reply:
We can, but why do you think we need it ?

The version file is part of distribution ( ie will be always there )